### ISSUE TITLE
cf create-space uses wrong space-quota

### ISSUE THREAD

**Issue by Zteve** [cloudfoundry/cli#775]
### Command
`cf create-space new-space-3 -o new-org -q default`
### CLI Version
cf version 6.15.0+fa1bfe2-2016-01-13
### Error
```
FAILED
Server error, status code: 400, error code: 40001, message: The app space info is invalid: space_quota_definition invalid_organization
FAILED
Server error, status code: 400, error code: 40001, message: The app space info is invalid: space_quota_definition invalid_organization
```
Even though the space-quota definition `default` exists in the current organization (`integration-test`) and the organization in which the new space is being created (`new-org`). The error is (apparently) that the `default` space quota definition is looked-up in the `integration-test` organization, and the guid for  that definition is used when creating the space.  Instead the guid for the quota definition in the new space's organization should be used.
### Stack Trace
N/A
### CC API Version
microPCF running API version 2.43.0
### CF Trace
```
Creating space new-space-3 in org new-org as admin...

REQUEST: [2016-02-23T14:41:14Z]
GET /v2/organizations/df6486e2-e31c-4223-80d1-d04e6fb493d1/space_quota_definitions HTTP/1.1

RESPONSE: [2016-02-23T14:41:14Z]
HTTP/1.1 200 OK

{
  "total_results": 1,
  "total_pages": 1,
  "prev_url": null,
  "next_url": null,
  "resources": [
    {
      "metadata": {
        "guid": "e8999f60-7507-4a60-acec-43431d34edc1",
        "url": "/v2/space_quota_definitions/e8999f60-7507-4a60-acec-43431d34edc1",
        "created_at": "2016-02-23T13:57:58Z",
        "updated_at": null
      },
      "entity": {
        "name": "default",
        "organization_guid": "df6486e2-e31c-4223-80d1-d04e6fb493d1",
        "non_basic_services_allowed": false,
        "total_services": 0,
        "total_routes": 0,
        "memory_limit": 2048,
        "instance_memory_limit": 1024,
        "app_instance_limit": -1,
        "organization_url": "/v2/organizations/df6486e2-e31c-4223-80d1-d04e6fb493d1",
        "spaces_url": "/v2/space_quota_definitions/e8999f60-7507-4a60-acec-43431d34edc1/spaces"
      }
    }
  ]
}

REQUEST: [2016-02-23T14:41:14Z]
GET /v2/organizations?q=name%3Anew-org&inline-relations-depth=1 HTTP/1.1

RESPONSE: [2016-02-23T14:41:14Z]
HTTP/1.1 200 OK

{
  "total_results": 1,
  "total_pages": 1,
  "prev_url": null,
  "next_url": null,
  "resources": [
    {
      "metadata": {
        "guid": "e26666fe-3096-402d-9680-c40f079eb054",
        "url": "/v2/organizations/e26666fe-3096-402d-9680-c40f079eb054",
        "created_at": "2016-02-23T13:26:48Z",
        "updated_at": null
      },
      "entity": {
        "name": "new-org",
        "billing_enabled": false,
        "quota_definition_guid": "e8705d39-0084-47f6-9ce1-7e08b79f8cf4",
        "status": "active",
        "quota_definition_url": "/v2/quota_definitions/e8705d39-0084-47f6-9ce1-7e08b79f8cf4",
        "quota_definition": {
          "metadata": {
            "guid": "e8705d39-0084-47f6-9ce1-7e08b79f8cf4",
            "url": "/v2/quota_definitions/e8705d39-0084-47f6-9ce1-7e08b79f8cf4",
            "created_at": "2016-02-23T09:24:06Z",
            "updated_at": null
          },
          "entity": {
            "name": "default",
            "non_basic_services_allowed": true,
            "total_services": -1,
            "total_routes": 1000,
            "total_private_domains": -1,
            "memory_limit": 102400,
            "trial_db_allowed": false,
            "instance_memory_limit": -1,
            "app_instance_limit": -1
          }
        },
        "spaces_url": "/v2/organizations/e26666fe-3096-402d-9680-c40f079eb054/spaces",
        "spaces": [
          {
            "metadata": {
              "guid": "b2cfaae7-d832-490b-ae8b-090b6b6b901c",
              "url": "/v2/spaces/b2cfaae7-d832-490b-ae8b-090b6b6b901c",
              "created_at": "2016-02-23T13:32:37Z",
              "updated_at": null
            },
            "entity": {
              "name": "new-space",
              "organization_guid": "e26666fe-3096-402d-9680-c40f079eb054",
              "space_quota_definition_guid": null,
              "allow_ssh": true,
              "organization_url": "/v2/organizations/e26666fe-3096-402d-9680-c40f079eb054",
              "developers_url": "/v2/spaces/b2cfaae7-d832-490b-ae8b-090b6b6b901c/developers",
              "managers_url": "/v2/spaces/b2cfaae7-d832-490b-ae8b-090b6b6b901c/managers",
              "auditors_url": "/v2/spaces/b2cfaae7-d832-490b-ae8b-090b6b6b901c/auditors",
              "apps_url": "/v2/spaces/b2cfaae7-d832-490b-ae8b-090b6b6b901c/apps",
              "routes_url": "/v2/spaces/b2cfaae7-d832-490b-ae8b-090b6b6b901c/routes",
              "domains_url": "/v2/spaces/b2cfaae7-d832-490b-ae8b-090b6b6b901c/domains",
              "service_instances_url": "/v2/spaces/b2cfaae7-d832-490b-ae8b-090b6b6b901c/service_instances",
              "app_events_url": "/v2/spaces/b2cfaae7-d832-490b-ae8b-090b6b6b901c/app_events",
              "events_url": "/v2/spaces/b2cfaae7-d832-490b-ae8b-090b6b6b901c/events",
              "security_groups_url": "/v2/spaces/b2cfaae7-d832-490b-ae8b-090b6b6b901c/security_groups"
            }
          },
          {
            "metadata": {
              "guid": "398a4236-c8e4-464e-ae1c-3ff98249e57c",
              "url": "/v2/spaces/398a4236-c8e4-464e-ae1c-3ff98249e57c",
              "created_at": "2016-02-23T14:24:20Z",
              "updated_at": null
            },
            "entity": {
              "name": "new-space-2",
              "organization_guid": "e26666fe-3096-402d-9680-c40f079eb054",
              "space_quota_definition_guid": null,
              "allow_ssh": true,
              "organization_url": "/v2/organizations/e26666fe-3096-402d-9680-c40f079eb054",
              "developers_url": "/v2/spaces/398a4236-c8e4-464e-ae1c-3ff98249e57c/developers",
              "managers_url": "/v2/spaces/398a4236-c8e4-464e-ae1c-3ff98249e57c/managers",
              "auditors_url": "/v2/spaces/398a4236-c8e4-464e-ae1c-3ff98249e57c/auditors",
              "apps_url": "/v2/spaces/398a4236-c8e4-464e-ae1c-3ff98249e57c/apps",
              "routes_url": "/v2/spaces/398a4236-c8e4-464e-ae1c-3ff98249e57c/routes",
              "domains_url": "/v2/spaces/398a4236-c8e4-464e-ae1c-3ff98249e57c/domains",
              "service_instances_url": "/v2/spaces/398a4236-c8e4-464e-ae1c-3ff98249e57c/service_instances",
              "app_events_url": "/v2/spaces/398a4236-c8e4-464e-ae1c-3ff98249e57c/app_events",
              "events_url": "/v2/spaces/398a4236-c8e4-464e-ae1c-3ff98249e57c/events",
              "security_groups_url": "/v2/spaces/398a4236-c8e4-464e-ae1c-3ff98249e57c/security_groups"
            }
          }
        ],
        "domains_url": "/v2/organizations/e26666fe-3096-402d-9680-c40f079eb054/domains",
        "domains": [
          {
            "metadata": {
              "guid": "10e30fe3-82e8-4967-9838-54a4462bd097",
              "url": "/v2/domains/10e30fe3-82e8-4967-9838-54a4462bd097",
              "created_at": "2016-02-23T09:24:06Z",
              "updated_at": "2016-02-23T09:24:06Z"
            },
            "entity": {
              "name": "local.micropcf.io",
              "router_group_guid": null
            }
          }
        ],
        "private_domains_url": "/v2/organizations/e26666fe-3096-402d-9680-c40f079eb054/private_domains",
        "private_domains": [

        ],
        "users_url": "/v2/organizations/e26666fe-3096-402d-9680-c40f079eb054/users",
        "users": [
          {
            "metadata": {
              "guid": "50c3719b-90d1-4af3-85ff-400500444c9b",
              "url": "/v2/users/50c3719b-90d1-4af3-85ff-400500444c9b",
              "created_at": "2016-02-23T09:24:09Z",
              "updated_at": null
            },
            "entity": {
              "admin": false,
              "active": true,
              "default_space_guid": null,
              "spaces_url": "/v2/users/50c3719b-90d1-4af3-85ff-400500444c9b/spaces",
              "organizations_url": "/v2/users/50c3719b-90d1-4af3-85ff-400500444c9b/organizations",
              "managed_organizations_url": "/v2/users/50c3719b-90d1-4af3-85ff-400500444c9b/managed_organizations",
              "billing_managed_organizations_url": "/v2/users/50c3719b-90d1-4af3-85ff-400500444c9b/billing_managed_organizations",
              "audited_organizations_url": "/v2/users/50c3719b-90d1-4af3-85ff-400500444c9b/audited_organizations",
              "managed_spaces_url": "/v2/users/50c3719b-90d1-4af3-85ff-400500444c9b/managed_spaces",
              "audited_spaces_url": "/v2/users/50c3719b-90d1-4af3-85ff-400500444c9b/audited_spaces"
            }
          }
        ],
        "managers_url": "/v2/organizations/e26666fe-3096-402d-9680-c40f079eb054/managers",
        "managers": [
          {
            "metadata": {
              "guid": "50c3719b-90d1-4af3-85ff-400500444c9b",
              "url": "/v2/users/50c3719b-90d1-4af3-85ff-400500444c9b",
              "created_at": "2016-02-23T09:24:09Z",
              "updated_at": null
            },
            "entity": {
              "admin": false,
              "active": true,
              "default_space_guid": null,
              "spaces_url": "/v2/users/50c3719b-90d1-4af3-85ff-400500444c9b/spaces",
              "organizations_url": "/v2/users/50c3719b-90d1-4af3-85ff-400500444c9b/organizations",
              "managed_organizations_url": "/v2/users/50c3719b-90d1-4af3-85ff-400500444c9b/managed_organizations",
              "billing_managed_organizations_url": "/v2/users/50c3719b-90d1-4af3-85ff-400500444c9b/billing_managed_organizations",
              "audited_organizations_url": "/v2/users/50c3719b-90d1-4af3-85ff-400500444c9b/audited_organizations",
              "managed_spaces_url": "/v2/users/50c3719b-90d1-4af3-85ff-400500444c9b/managed_spaces",
              "audited_spaces_url": "/v2/users/50c3719b-90d1-4af3-85ff-400500444c9b/audited_spaces"
            }
          }
        ],
        "billing_managers_url": "/v2/organizations/e26666fe-3096-402d-9680-c40f079eb054/billing_managers",
        "billing_managers": [

        ],
        "auditors_url": "/v2/organizations/e26666fe-3096-402d-9680-c40f079eb054/auditors",
        "auditors": [

        ],
        "app_events_url": "/v2/organizations/e26666fe-3096-402d-9680-c40f079eb054/app_events",
        "space_quota_definitions_url": "/v2/organizations/e26666fe-3096-402d-9680-c40f079eb054/space_quota_definitions",
        "space_quota_definitions": [
          {
            "metadata": {
              "guid": "df09b9df-c62c-41ae-98ca-f56f0ffb7c4a",
              "url": "/v2/space_quota_definitions/df09b9df-c62c-41ae-98ca-f56f0ffb7c4a",
              "created_at": "2016-02-23T14:19:04Z",
              "updated_at": null
            },
            "entity": {
              "name": "default",
              "organization_guid": "e26666fe-3096-402d-9680-c40f079eb054",
              "non_basic_services_allowed": false,
              "total_services": 0,
              "total_routes": 0,
              "memory_limit": 2048,
              "instance_memory_limit": 1024,
              "app_instance_limit": -1,
              "organization_url": "/v2/organizations/e26666fe-3096-402d-9680-c40f079eb054",
              "spaces_url": "/v2/space_quota_definitions/df09b9df-c62c-41ae-98ca-f56f0ffb7c4a/spaces"
            }
          }
        ]
      }
    }
  ]
}

REQUEST: [2016-02-23T14:41:14Z]
POST /v2/spaces?async=true&inline-relations-depth=1 HTTP/1.1

{"name":"new-space-3","organization_guid":"e26666fe-3096-402d-9680-c40f079eb054","space_quota_definition_guid":"e8999f60-7507-4a60-acec-43431d34edc1"}

RESPONSE: [2016-02-23T14:41:14Z]
HTTP/1.1 400 Bad Request

{
  "code": 40001,
  "description": "The app space info is invalid: space_quota_definition invalid_organization",
  "error_code": "CF-SpaceInvalid"
}

FAILED
Server error, status code: 400, error code: 40001, message: The app space info is invalid: space_quota_definition invalid_organization
FAILED
Server error, status code: 400, error code: 40001, message: The app space info is invalid: space_quota_definition invalid_organization
```
### Platform Details
Mac OS X El Capitan Version 10.11.3
### Shell
Terminal

**Comment by dkoper** [comment]
Hi @Zteve 

Thank you for reporting this issue. I believe your conclusion is on the ball. The team will take a look.

Cheers,
Dies Koper
CF CLI PM

**Comment by dkoper** [comment]
This has been fixed in cf CLI v6.17.0.
https://github.com/cloudfoundry/cli#downloads

Thanks again!
Dies Koper
CF CLI PM
